e2e/brickmux_test.go

```diff
@@ -400,7 +402,7 @@ func TestBrickMux(t *testing.T) {
 			index = index + 1
 		}
 		// Wait for added bricks' glusterfsds to Sign In
-		time.Sleep(5 * time.Millisecond)
+		time.Sleep(70 * time.Millisecond)
```

Comment:
@vpandey-RH will this timeout will behave differently in different environments based on the network?

Reply:
@Madhu-1 I opted for this timeout just to be safe that the test should pass almost everytime. I am still retesting this and would like to keep trigerring tests atleast for today.